### PR TITLE
Fix is_conditional bug

### PR DESCRIPTION
**Pull request by smcmurtry**
# Summary | Résumé

Closes https://github.com/cds-snc/notification-planning/issues/847

This PR fixes a bug in #128 where the `{"is_conditional": True}` property can be overwritten to `False` if the same variable is used multiple times in a template. The desired behaviour is that, if a variable is used in a conditional statement, we should guide the user towards setting it to a True/False value and always display the radio buttons instead of free text in the "send to one recipient" flow here: https://github.com/cds-snc/notification-admin/pull/1316

### CODE REVIEW

**Comment by smcmurtry** [comment]
@andrewleith For this bug fix I modified the notification-utils code installed in the notification-admin dev container with the changes in this PR. That is probably the easiest way to review.

**Review by amazingphilippe — APPROVED** [review]
This looks good!

File: tests/test_field.py [hunk 1]
```diff
@@ -272,3 +272,18 @@ def test_what_will_trigger_conditional_placeholder(value):
...
     assert str(Field("list: ((placeholder))", values, markdown_lists=True)) == expected_as_markdown
     assert str(Field("list: ((placeholder))", values)) == expected
+
+
+@pytest.mark.parametrize(
+    "template_str, result",
+    [
+        ("Lorem ipsum ((var))", False),
+        ("Lorem ipsum ((var??Conditional text))", True),
+        ("Lorem ipsum ((var)) Lorem ipsum ((var??Conditional text)) Lorem ipsum ((var))", True),
```
Comment (andrewleith) [inline tests/test_field.py:282]: Maybe we could add 2 more cases to test:
1. Lorem ipsum ((var)) Lorem ipsum ((var??Conditional text)) 
2. Lorem ipsum ((var??Conditional text)) Lorem ipsum ((var))

File: notifications_utils/field.py [hunk 2]
```diff
@@ -167,10 +167,13 @@ def placeholders(self):
 
     @property
     def placeholders_meta(self):
-        meta = {
-            Placeholder(body).name: {"is_conditional": Placeholder(body).is_conditional()}
-            for body in re.findall(self.placeholder_pattern, self.content)
-        }
+        meta = {}
+        for body in re.findall(self.placeholder_pattern, self.content):
```
Comment (jimleroyer) [inline notifications_utils/field.py:171]: Looks good. A summary comment to explain why these two conditions are necessary would be nice on our selves.

File: tests/test_field.py (hunk 1 above)
Comment (smcmurtry) [reply tests/test_field.py:282]: Thanks, added!

File: notifications_utils/field.py (hunk 2 above)
Comment (smcmurtry) [reply notifications_utils/field.py:175]: 👍

File: notifications_utils/version.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
-__version__ = "48.1.1"
+__version__ = "48.1.2"
```
Comment (andrewleith) [inline notifications_utils/version.py:1]: I bumped utils to 48.2.0 for the lang tag fixes so you will need to bump it again here :)

**Pull request merged**